static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -10,7 +10,7 @@ import {trackAnalytics} from 'sentry/utils/analytics';
 import useOrganization from 'sentry/utils/useOrganization';
 import useProjectFromSlug from 'sentry/utils/useProjectFromSlug';
 
-import type {TimelineEvent, TimelineIssuePlatformEvent} from './useTraceTimelineEvents';
+import type {TimelineEvent} from './useTraceTimelineEvents';
```

Comment:
We do not need to do this anymore:
`-      const issuePlatformEvent = event as TimelineIssuePlatformEvent;`